main.js

```diff
@@ -2,3 +2,33 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+
+  for (let i=0 ; i <5;i++){
+
+    const img = document.createElement('img');
+    const apiUrl = `https://rickandmortyapi.com/api/character/${i}`;
+
+    axios(
+        {
+            method: 'GET',
+            url: apiUrl
+           
+        }
+    )
+        .then(
+            ressponds => {
+                const imgUrl = ressponds.data.image;
+                img.src = imgUrl;
```

Comment:
I think you forget to create the image.
img is undefined.
Please resolve the homework and open another pull request .
```suggestion
                img.src = imgUrl;
```